src/data/README.md

```diff
@@ -18,7 +18,234 @@ TODO - describe the properties used in DevDot
+
+Example: `h2` with `300` size:
+
+```json5
+{
+  "type": "heading",
+  "heading": "Featured Reference Docs",
+  "level": 2,
+  "size": 300,
+}
+```
+
+![](https://user-images.githubusercontent.com/4624598/158818745-a20d1892-efa6-4053-9d00-811645d642aa.png)
+
+Example: `h2` with `400` size:
+
+```json5
+{
+  "type": "heading",
+  "heading": "Explore Waypoint Documentation",
+  "level": 2,
+  "size": 400,
+}
+```
+
+![](https://user-images.githubusercontent.com/4624598/158818617-2b8ce029-ad41-4081-8701-869d51abf40b.png)
+
+</details>
+
+<!-- block type: get_started -->
+
+<details>
+<summary>Block type: <code>"get_started"</code></summary>
+
+The `"get_started"` block renders a heading, descriptive text, and a single link alongside a product icon.
+
+| Property | Type | Details |
+| --- | --- | --- |
+| `type` | `"get_started"` | Block type |
+| `product` | [ProductSlug](../types/products.ts) | Product icon to be shown. |
+| `heading` | `string` | Text for the heading |
+| `text` | `string` | Descriptive text shown below the heading |
+| `link` | `{ text: string, url: string }` | [`StandaloneLink`](../components/standalone-link/index.tsx) shown below the body text |
+
+Example: Waypoint `"get_started"` block
+
+```json5
+{
+  "type": "get_started",
+  "product": "waypoint",
+  "heading": "Introduction to Waypoint",
+  "text": "Welcome to Waypoint! This introduction section covers what Waypoint is, the problem Waypoint aims to solve, and how Waypoint compares to other software.",
+  "link": {
+    "text": "Get Started",
+    "url": "/waypoint/docs/intro"
+  }
+}
+```
+
+![](https://user-images.githubusercontent.com/4624598/158821262-03798dca-12e6-487b-ac3e-e8bab51be8b1.png)
+
+</details>
+
+<!-- block type: cards -->
+
+<details>
+<summary>Block type: <code>"cards"</code></summary>
+
+The `"cards"` block displays a grid of `CardLink`s, each linked using a single `url`.
+
+| Property | Type | Details |
+| --- | --- | --- |
+| `type` | `"cards"` | Block type |
+| `columns` | `2 \| 3` | The maximum number of columns |
+| `cards` | `Array<{ icon, iconBrandColor, heading, text, url, tags }>` | An array of objects, described in detail below |
+
+Each item in the `cards` array has the following structure:
+
+| Property | Type | Details |
+| --- | --- | --- |
+| `icon` | (optional) `string` | Optional icon to show at the top of the card. Must be one of the keys in [the card component's icon dictionary](../views/product-landing/components/cards/icon-dict.js) |
+| `iconBrandColor` | (optional) [Products](https://github.com/hashicorp/web-platform-packages/blob/69b2cf609a254c0f469142266ecb60ffe04cbc7a/packages/product-meta/index.tsx#L11) string or `"neutral" \| "neutral-dark"` | Optional brand color override to apply to the icon. Defaults to the current product context. |
```

Comment:
We should link to `ProductSlug` type here since it's built on top of what you linked, and adds `sentinel` and `hcp` slugs.

```suggestion
| `iconBrandColor` | (optional) [ProductSlug](/src/types/products.ts) string or `"neutral" \| "neutral-dark"` | Optional brand color override to apply to the icon. Defaults to the current product context. |
```

Reply:
I think `Products` is the correct type here - passing `"hcp"` or `"sentinel"` might not work as expected, since they're not supported options in `IconTile`. Maybe more clear if I link to [`ProductBrandColor` within `IconTile`'s `types.ts` file](https://github.com/hashicorp/dev-portal/blob/476bd48cbd0b28bcbe86d45af758cbbad66ee807/src/components/icon-tile/types.ts#L3), which is equivalent but declared within the project and more direct (since this is ultimately getting passed to `IconTile`). I just made that update, let me know what you think!

Reply:
Ah okay! I misremembered whether _IconTile_ handled those two or not. But I like your idea of linking to `ProductBrandColor` even more! 👏🏻